src/pages/Facility/locations/Utils.tsx

```diff
@@ -0,0 +1,13 @@
+import { LocationList as LocationListType } from "@/types/location/location";
+
+export function getParentChain(location: LocationListType): LocationListType[] {
```

Comment:
we could make this more generic such that anything that has a `parent` attribute, should work.

That way this would also work for organization objects as well for example.
```suggestion
export function getParentChain(location: LocationListType): LocationListType[] {
```

Reply:
don't need it so deleted the removed the usage from devicelist